src/main/webapp/data/userMapData.json

```diff
@@ -49,12 +49,12 @@
       "Photography Studies College",
       "Queensland University of Technology",
       "RMIT University",
+      "St. Scholastica's College",
+      "Star of the Sea Catholic College",
       "St Catherine's Catholic College",
       "St Ives High School",
       "St Joseph's School Northam",
       "St Paul's Anglican Grammar School",
-      "St. Scholastica's College",
-      "Star of the Sea Catholic College",
```

Comment:
This should be back above St Catherine's, since 'a' in "star" is before 'c' in "Catherine"